src/modules/addList.js

```diff
@@ -0,0 +1,134 @@
+export const form = document.querySelector('form');
+export const newTaskEl = document.getElementById('new-task');
+export const toDoListEl = document.querySelector('.toDoList');
+export const btnTask = document.querySelector('.btn-task');
+export const clearAll = document.querySelector('.clear-all');
+export const inputEl = document.querySelectorAll('.inputEl');
+let editId;
+let isEdited = false;
+let toDoList = [];
+export default class Tasks {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  renderTasks() {
+    return `
+         <li class="task">
+            <label for="task-${this.index}">
+                <input class="updateStat" type="checkbox" id="${this.index}">
+                <p>${this.description}</p>
+             </label>
+            <div class="setting">
+                <i class="fas fa-ellipsis-v" class="fa-ellipsis-v"></i>
+                <ul class="task-menu">
+                    <li><i class="fas fa-pen" id="${this.index}"></i></li>
+                    <li><i class="fa fa-trash" id='${this.index}' aria-hidden="true"></i></li>
+                </ul>
+            </div> 
+         </li>
+          `;
+  }
+
+  static appendTask(task) {
+    let index = 1;
+    if (toDoList.length >= 1) {
+      index = toDoList[toDoList.length - 1].index + 1;
+    }
+    task.index = index;
+    toDoList.push(task);
+    localStorage.setItem('ourTasks', JSON.stringify(toDoList));
+  }
+
+  static removeTask(index) {
+    toDoList = toDoList.filter((task) => task.index !== Number(index));
+    localStorage.setItem('ourTasks', JSON.stringify(toDoList));
+  }
+}
+export const listFromLocalStorage = JSON.parse(
+  localStorage.getItem('ourTasks'),
+);
+export function displayTask() {
+  // eslint-disable-next-line max-len
```

Comment:
Please ensure that linters are not disabled. A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).